salt/scripts.py

```diff
@@ -107,11 +107,6 @@ def salt_minion():
     if '' in sys.path:
         sys.path.remove('')
 
-    if salt.utils.is_windows():
-        minion = salt.cli.daemons.Minion()
-        minion.start()
-        return
-
     if '--disable-keepalive' in sys.argv:
```

Comment:
@rallytime Did you remove the `if salt.utils.is_windows()` block? It needs to stay in there for 2016.3